modules/sdk-core/package.json

```diff
@@ -37,15 +37,21 @@
   "dependencies": {
     "@bitgo/bls-dkg": "^1.0.2",
     "@bitgo/statics": "^6.17.0",
+    "@bitgo/utxo-lib": "^2.2.2",
     "@stablelib/hex": "^1.0.0",
+    "big.js": "^3.1.3",
     "bignumber.js": "^9.0.0",
     "bip32": "^2.0.6",
     "bitcoinjs-lib": "npm:@bitgo/bitcoinjs-lib@6.1.0-rc.3",
+    "bitcoinjs-message": "^2.0.0",
     "bs58": "^4.0.1",
     "create-hmac": "^1.1.7",
+    "debug": "^3.1.0",
+    "libsodium-wrappers-sumo": "^0.7.9",
```

Comment:
same here. Remove this from bitgo's package.json

Reply:
I think `libsodium-wrappers-sumo` is from account-lib's package.json. I'm removing it now.